src/pages/main_layout.cr

```diff
@@ -20,36 +20,198 @@ abstract class MainLayout
+
+          div class: "my-10 md:mt-10 md:mb-8" do
+            link "View on GitHub", "https://github.com/luckyframework/lucky", class: "btn btn--blue w-full sm:w-auto sm:mr-5"
+            link "Get Started", Guides::GettingStarted::Installing, class: "btn w-full sm:w-auto mt-6 sm:mt-0"
           end
         end
-        render_main_content
       end
     end
   end
 
-  private def render_main_content
-    div class: "container mx-auto px-6" do
+  private def render_feature_grid
+    div class: "container text-center md:text-left mx-auto px-6 py-5 bg-white md:bg-transparent border-b border-grey-light md:border-none" do
       mount Shared::FlashMessages.new(@context.flash)
-      # TODO: Add real page content later
-      # content
+      div class: "flex flex-col md:flex-row md:bg-white mx-auto md:rounded-lg md:shadow md:-mt-12 md:px-5 md:py-12" do
+        content_block "Say goodbye to slow", <<-TEXT
+        A simple hello world response is returned in 0.1ms. Rendering
+        complex JSON usually takes only a few milliseconds.
+        TEXT
+
+        content_block "Batteries included", <<-TEXT
+        Authentication, asset management, CORS, database ORM, and more can
+        all be included when creating a new Lucky project.
+        TEXT
+
+        content_block "Fewer bugs. More joy", <<-TEXT
+        Instead of finding bugs in QA or in production, Lucky is designed to
+        catch as many bugs as possible at compile time.
+        TEXT
+      end
+    end
+  end
+
+  private def content_block(title, body)
+    div class: "w-full md:w-1/3 sm:px-0 md:px-8 my-5 md:my-0" do
+      h3 title, class: "mb-3 text-black font-bold text-base"
+      para body, class: "leading-loose text-sm text-grey-darker"
+    end
+  end
+
+  private def render_freeform_text
+    render_markdown_content <<-TEXT
+    ## Why is modern web development so complicated?
+
+    **There are more choices in tech than ever**, and with it comes more questions and more complexity.
+
+    Have you ever thought:
+
+    * "Why do I have to write **so much boilerplate**?"
+    * "How do **so many bugs make it to production**?"
+    * "Why is my app **so slow**!?"
+    * "Why do I have to choose between fast prototyping and long term maintainability?"
+
+    ## What if you could get something up quickly *and* maintain it for years to come?
+
+    Imagine this:
+
+    * Generators for quick code creation
+    * Built in authentication (Optional)
+    * Secure by default
+    * Built in Webpack config to get a SPA off the ground fast (Optional)
+    * Insanely fast execution so users aren’t left waiting
+    * Powerful type system that helps catch bugs before they ever hit production
+
+    If this sounds interesting, [let's get started](#{Guides::GettingStarted::Installing.path}).
+
+    If you're not quite ready to dive in but want to stay up to date,
+    [subscribe to our blog](#{Blog::Index.path}), or [follow us on
+    Twitter](https://twitter.com/luckyframework).
```

Comment:
I decided to just remove the "Is Lucky right for me?" since I think it wasn't that helpful. LMK if you feel differently though. For now I've just added this little blurb to take its place